qcodes/instrument_drivers/stanford_research/SR830.py

```diff
@@ -472,6 +472,60 @@ def parse_offset_get(s):
         self._buffer2_ready = False
 
         self.connect_message()
+    
+    
+    SNAP_PARAMETERS = {
+            'x': '1',   
+            'y': '2',  
+            'r': '3', 
+            'p': '4',   
+        'phase': '4',  
+           'θ' : '4',
+        'aux 1': '5',  
+        'aux 2': '6',   
+        'aux 3': '7',  
+        'aux 4': '8',  
+         'freq': '9',   
+         'ch 1': '10',
+         'ch 2': '11'  
+    }
+    
+    def snap(self, *parameter_names: str) -> Tuple[float, ...]:
+            """
+            Gets the values of either 2, 3, 4, 5 or 6 parameters at a single instant.
+            For example, SNAP? is a way to query values of
+            X and Y (or R and θ) which are taken at the same time. This is important
+            when the time constant is very short. Using the OUTP? or OUTR? com-
```

Comment:
within qcodes, we are generally following pep8, which requires the lines of code to not exceed 80 characters. Could you please adjust the code accordingly?

Reply:
Yep. This can be done.